src/engine/features.rs

```diff
@@ -19,7 +19,7 @@ lazy_static! {
         CREATE_REMOTE_PRIVATE,
         CHECK_EXISTS,
         MOVE_REMOTE,
-        FORK_REPOSITORY,
+        FORK_REMOTE,
```

Comment:
Should we add a default to `FeaturesBuilder::with_defaults` for this so that it's enabled for all users without configuration?